### ISSUE TITLE
switch/swapp   left<->right source places

### ISSUE THREAD

**Issue by valentijnpeters** [nokia/RED#406]
on the left I have opened a resource.robot on the right I have opened a test.robot.
I'd like them to show the other way around (resource left and test right)

there is no button to easily swapp  those files from places.
<-->

(oh wait I guess that's an Eclipse thing then?)

**Comment by michalanglart** [comment]
Yeah, the way how editors are opened, docked and so one is handled by the eclipse.
